### PR TITLE
fix(sidebar): unavailable items from US customers

### PR DESCRIPTION
**Pull request by antleblanc**

### CODE REVIEW

File: client/components/sidebar/sidebar.config.js [hunk 1]
```diff
@@ -23,9 +23,11 @@ angular.module("managerApp").config(function (SidebarMenuProvider) {
     }
     /*----------  SERVICES MENU ITEMS  ----------*/
     function initSidebarMenuItems (services, locale) {
-        IaasSectionSidebarService.fillSection(services.iaas);
-        PaasSectionSidebarService.fillSection(services.paas);
-        MetricsSectionSidebarService.fillSection(services.metrics);
+        if (TARGET !== "US") {
```
Comment (Netnix) [inline client/components/sidebar/sidebar.config.js]: Do not use TARGET, use FeatureAvailability everywhere instead.

File: client/components/sidebar/sidebar.config.js [hunk 2]
```diff
@@ -74,23 +79,32 @@ angular.module("managerApp").config(function (SidebarMenuProvider) {
...
-        }, {
-            title: $translate.instant("cloud_sidebar_actions_menu_iplb"),
-            icon: "ovh-font ovh-font-ip",
-            href: URLS.website_order.load_balancer[locale],
-            target: "_blank",
-            external: true
-        }, {
+        });
+
+        if (TARGET !== "US") {
```
Comment (Netnix) [inline client/components/sidebar/sidebar.config.js]: Do not use TARGET, use FeatureAvailability everywhere instead.

**Comment by antleblanc** [comment]
@Netnix changes applied. thx for the review.

File: client/components/sidebar/sidebar.config.js [hunk 3]
```diff
@@ -125,6 +145,7 @@ angular.module("managerApp").config(function (SidebarMenuProvider) {
         SidebarMenu.loadDeferred.promise
             .then(function (data) {
                 initSidebarMenuActionsOptions(data.user.ovhSubsidiary);
+                console.log("data.products.results", data.products.results);
```
Comment (Netnix) [inline client/components/sidebar/sidebar.config.js]: 😝

Comment (antleblanc) [reply client/components/sidebar/sidebar.config.js]: Good catch! That's removed.

File: client/app/config/US/common.constants.js [hunk 4]
```diff
@@ -4,91 +4,91 @@ angular.module("managerApp")
...
         },
         website_order: {
             "cloud-resell-eu": {
                 US: projectName =>
-                    `https://www.ovh.us/order/express/#/express/review?products=~(~(planCode~'project-eu~productId~'cloud~quantity~1~duration~'P1M~configuration~(~(label~'description~values~(~'${encodeURIComponent(projectName)})))))`
+                    `https://ovhcloud.com/order/express/#/express/review?products=~(~(planCode~'project-eu~productId~'cloud~quantity~1~duration~'P1M~configuration~(~(label~'description~values~(~'${encodeURIComponent(projectName)})))))`
             },
             dedicated_server: {
-                US: "https://www.ovh.us/dedicated-servers/"
+                US: "https://ovh.us/order/dedicated/#/dedicated/select"
```
[the remark this answers is not in the document]
Comment (Jisay) [reply client/app/config/US/common.constants.js]: for now, no. Order are on ovh.us

**Comment by Jisay** [comment]
Closed due to #732

**Comment by antleblanc** [comment]
Thanks @Jisay to handle it during my days off:)

**Pull request closed without merging**